tests/multicolumn_map_expectations/expect_multicolumn_values_to_be_unique.json

```diff
@@ -0,0 +1,50 @@
+{
+  "expectation_type" : "expect_multicolumn_values_to_be_unique",
+  "datasets" : [{
+    "data" : {
+      "w" : [2, 3, 4, 5, 6, 7, 8, 9, 10, null],
+      "x" : [1, 2, 3, 4, 5, 6, 7, 8, 9, 10],
+      "y" : [2, 3, 4, 5, 6, 7, 8, 9, 10, null],
+      "z" : [1, 2, 3, 4, 5, null, null, null, null, null],
+      "a" : [1, 1, 1, 1, 1, 2, 2, 2, 2, null],
+      "b" : [1, 2, 1, 2, 1, 2, 1, 2, 1, null]
+    },
+    "tests" : [{
```

Comment:
For disambiguation sake, it would be nice to see a first example that used the default value for "ignore_row_if"

Reply:
The first one is using the default value, but you're right that it's still explicitly setting that. I've updated the test and its description to make that clearer.